solutions/swb-ui/infrastructure/package.json

```diff
@@ -17,38 +17,52 @@
   },
   "scripts": {
     "add-license-header": "license-check-and-add add -f license-add-config.json",
-    "build": "heft build --clean",
-    "build:test": "rushx build && rushx test",
+    "build": "heft build --clean && rushx pkg-json-lint",
+    "build:test": "heft test --clean && rushx pkg-json-lint && rushx make-badges",
     "cdk": "cdk",
     "cdk-deploy": "rushx build && cdk deploy --all --require-approval never --outputs-file ./src/config/${STAGE}.json",
+    "cdk:bootstrap": "cdk bootstrap",
+    "cdk:synth": "cdk synth",
     "check-license-header": "license-check-and-add check -f license-add-config.json",
+    "depcheck": "depcheck",
     "license-checker": "license-checker --onlyAllow 'MIT; Apache-2.0; ISC; BSD'",
+    "lint:fix": "eslint . --fix",
+    "make-badges": "istanbul-badges-readme --coverageDir=./temp/coverage --exitCode=1",
+    "pkg-json-lint": "npmPkgJsonLint -c ../../../.npmpackagejsonlintrc.json .",
     "sort-package-json": "sort-package-json package.json",
-    "test": "rushx test:only",
+    "test": "rushx test:only && rushx make-badges",
     "test:only": "heft test --clean --no-build",
     "watch": "tsc -w"
   },
   "dependencies": {
     "js-yaml": "^4.1.0"
   },
   "devDependencies": {
+    "@aws/eslint-config-workbench-core-eslint-custom": "workspace:*",
+    "@rushstack/eslint-config": "^3.0.0",
     "@rushstack/heft": "^0.47.5",
     "@rushstack/heft-jest-plugin": "^0.3.28",
     "@rushstack/heft-node-rig": "^1.10.5",
+    "@types/heft-jest": "1.0.2",
     "@types/jest": "^27.4.1",
     "@types/js-yaml": "^4.0.5",
     "@types/node": "^14",
     "@types/prettier": "2.6.3",
     "aws-cdk": "^2.12.0",
     "aws-cdk-lib": "^2.29.1",
+    "cdk-nag": "^2.12.25",
     "constructs": "^10.0.0",
-    "jest": "^27.5.1",
+    "depcheck": "^1.4.3",
+    "eslint": "^8.7.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-security": "^1.4.0",
+    "istanbul-badges-readme": "1.8.1",
     "license-check-and-add": "^4.0.5",
     "license-checker": "^25.0.1",
+    "npm-package-json-lint": "^6.3.0",
+    "npm-package-json-lint-config-default": "^5.0.0",
     "sort-package-json": "^1.57.0",
     "source-map-support": "^0.5.16",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.4.0",
```

Comment:
Removing these (and the jest above) breaks me locally

Reply:
@pbabz You shouldn't need these. pushing an update. Can you please try it out again